app/src/main/java/com/zendrive/zendrivesdkdemo/MainFragment.java

```diff
@@ -77,29 +91,34 @@ public void onClick(View v) {
                 Zendrive.teardown();
 
                 // Initialize zendrive sdk
-                String driverId = "someDriverId";
-                MainFragment.this.initializeZendriveSDK(driverId);
+                MainFragment.this.initializeZendriveSDK();
```

Comment:
Disable/Hide restartSDKButton if we don't have valid values for applicationKey/driverId.